bundles/org.openhab.binding.millheat/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+  xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+  <config-description uri="thing-type:millheat:account">
+    <parameter name="username" type="text">
+      <label>Username</label>
+      <description>Your Millheat app username (email)</description>
+      <required>true</required>
+    </parameter>
+
+    <parameter name="password" type="text">
+      <label>Password</label>
+      <description>Your Millheat app password</description>
+      <required>true</required>
+      <context>password</context>
+    </parameter>
+
+    <parameter name="refreshInterval" type="integer" required="false" min="30" unit="s">
+      <label>Refresh interval</label>
```

Comment:
Can you make sure that every word from this label starts with a capital this is (more or less) the default in ESH / openHAB.